amr-wind/actuator_line/NREL5MW_ALM_BD/setup/README.md

```diff
@@ -0,0 +1,277 @@
+0                      UAStartRad  - Starting radius for dynamic stall (fraction of rotor radius [0.0,1.0]) [used only when UA_Mod>0; if line is missing UAStartRad=0]
+1                      UAEndRad    - Ending radius for dynamic stall (fraction of rotor radius [0.0,1.0]) [used only when UA_Mod>0; if line is missing UAEndRad=1]
+```
+
+**Beamdyn changes**
+
+In addition to the changes listed above, we also remove one of the blade stations from the beamdyn blade definition files.  The second blade station is removed, so the total number of key points drops from 49 to 48.  This is also handled automatically through the `downloadOFmodel.py` step above.
+
+
+<!--
+```yaml
+  FSTFile:
+    DT: 0.0008608815426997245 # Typically 0.005, make smaller for beamdyn
+    DT_Out: 0.0215220385675
+    CompElast:  2  # use beamdyn
+    CompInflow: 2
+    OutFileFmt: 3
+    AirDens:    1.000
+  EDFile:
+    NacYaw:   30.0
+    YawDOF:   False
+  AeroFile:
+    Wake_Mod: 0
+```
+-->
+
+### ALM settings
+
+When using the actuator line method in AMR-Wind, several additional parameters need to be set for this case.  The number of actuator line points along the length of the blade and tower needs to be set, as well as the magnitude of epislon, which sets the amount of smearing required for the applied force at each actuator force point.  These settings are defined in through these parameters in the AMR-Wind input file:
+
+```
+Actuator.T0.num_points_blade             = 25                  
+Actuator.T0.num_points_tower             = 11                  
+Actuator.T0.epsilon                      = 1.5 1.5 1.5         
+Actuator.T0.epsilon_tower                = 1.5 1.5 1.5         
+```
+
+## Optional: using the jupyter notebook
+
+If you're interested in modifying the AMR-Wind input file, for instance, to change the refinement regions or sampling planes, you can use the Jupyter notebook [SetupALM_OFv402.ipynb](SetupALM_OFv402.ipynb).  This will allow you to visualize the loccations of the refinements or planes, and will generate the AMR-Wind input file [NREL5MW_ALM_BD_OFv402.inp](../input_files/NREL5MW_ALM_BD_OFv402.inp) at the end of the process.
+
+There are a few modifications required to execute the notebook.  First, the location of the AMR-Wind frontend directory (and the utilities subdirectory) should be provided in the `amrwindfedirs` list here: 
+```python
+# Add any possible locations of amr-wind-frontend here
+amrwindfedirs = ['/projects/wind_uq/lcheung/amrwind-frontend',
+                 '/projects/wind_uq/lcheung/amrwind-frontend/utilities',
+                ]
+```
+
+Secondly, the location of the precursor directory and the inputfile from the precursor run should be supplied in the `precursordir` and the `precursorsetup` directories, respectively.
```

Comment:
Is "precursorsetup" a directory?  Here it seems to point directly to the AMR-Wind input file.

Reply:
Oh yeah, that was ambiguous.  Fixed that to refer to it as the input file.